packages/manager/modules/cloud-universe-components/src/currency/constants.js

```diff
@@ -0,0 +1,38 @@
+export const CURRENCY_SUBSIDIARY = {
```

Comment:
I know this is not the point of this PR but https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat